### ISSUE TITLE
SharePoint - CreateGroupForSite: "Modern site creation is not allowed. Contact your administrator for more information."

### ISSUE THREAD

**Issue by fzbm** [MicrosoftDocs/microsoft-365-docs#10092]
Since about a week it seems like it is no longer possible to connect a SharePoint site to a group using the SharePoint REST API endpoint `_api/GroupSiteManager/CreateGroupForSite`. Every call with an access token for a user which is allowed to create groups fails with the following response. The endpoint worked flawlessly until the previously mentioned week ago.

```json
{
    "error": {
        "code": "-2146232832, Microsoft.SharePoint.SPException",
        "message": "Modern site creation is not allowed. Contact your administrator for more information."
    }
}
```

The problem occurs on two tenants and both have the property `EnableGroupCreation` set to `false` and `GroupCreationAllowedGroupId` set to a valid group ID. The account we use to call the endpoint is a member of those groups.

It looks like we are not the only ones with this problem: https://www.reddit.com/r/sysadmin/comments/y9mfjm/sharepoint_online_can_no_longer_create_sites/

Has something changed? We use the PnP.Framework implementation of it and there is no recent change in the PnP.Core and PnP.Framework repositories regarding this.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 445c0a2e-1a22-d90f-0c73-0edf34f2b99a
* Version Independent ID: d534ac69-134d-0198-fcb9-e19d1ed44d4b
* Content: [Manage who can create Microsoft 365 Groups](https://learn.microsoft.com/en-us/microsoft-365/solutions/manage-creation-of-groups?view=o365-worldwide)
* Content Source: [microsoft-365/solutions/manage-creation-of-groups.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/solutions/manage-creation-of-groups.md)
* Service: **o365-solutions**
* GitHub Login: @MikePlumleyMSFT
* Microsoft Alias: **mikeplum**

**Comment by msbemba** [comment]
@fzbm Thank you for your feedback. This sounds more like a product bug and I'd recommend you contacting support team via the need help section on the admin center, so that a support engineer can help fix this or raise a bug with the product team.

Hope this helps!

Thanks
Sri

**Comment by yogkumgit** [comment]
@fzbm  Thank you very much for your kind feedback. Considering your valuable feedback, please log this issue as advised. Here we are limited to documentation fixes and improvements only. Concerning team will evaluate and take the appropriate action, as per their schedule. We proceed here to close it. Thanks again for taking out some time and sharing your feedback.

**Comment by fzbm** [comment]
Hi @msbemba and @yogkumgit,

thanks for your fast responses. There was a holiday yesterday in my country and I additionally took a day off to extend my weekend by two days. Therefore, the late response from my site.

Anyway ... I tested it again and it looks like the problem fixed "itself". 😉 Sadly, I cannot find an entry about this in the message center, or the problem was fixed during fixing another issue mentioned in the center.
